### DISCUSSION TITLE
Auto Enable Darkmode

### DISCUSSION THREAD

**Discussion by daes10** [Hypfer/Valetudo#1774] [Q&A / Support] [answered]
Hello all together,

i want to know, if there is a way to "auto" enable darkmode in the ui. I embeded the Valetudo UI in my homeautomation as an iframe. It shows up on the Wall in a Tablet. The problem is, when it's reloading, every time i must reset the darkmode slider. (because my Website has a darktheme and it's all the same color ;)  )

Is there a way to permanently leave it on darkmode? Or is it possible to call a custom url (that i can call in my embeded iframe) to open it in darkmode?

If this is not possible, i think it would by easy to implement some kind of Setting to choose the personal preference.

And every time, thanks on all people contribute to this project! It's an awesome project that i use **every** day!

**Comment by Hypfer** [comment, marked as answer]
Ah yes, we had that issue before.

Check your browsers privacy settings and or any content blocker add-ons.
You're likely blocking third-party cookies which prevents this setting from being saved when the frontend is shown as an iframe.

**Reply by daes10** [reply]
I searched for a darkmode question before, but don't found one last time. 
But thanks a lot for the answer, it worked well as described!
